src/events/common.ts

```diff
@@ -39,18 +39,41 @@ export class AppDiscord {
   async isReady([client]: ArgsOf<'ready'>) {
     const channelIds = JSON.parse(process.env.CHANNELS)
 
-    const devChannel = client.channels.cache.get(channelIds.DEV) as TextChannel
-    const prodChannel = client.channels.cache.get(
-      channelIds.PROD,
+    const devWikiChannel = client.channels.cache.get(
+      channelIds.DEV.WIKI,
+    ) as TextChannel
+    const devHiiqChannel = client.channels.cache.get(
+      channelIds.DEV.HIIQ,
+    ) as TextChannel
+    const prodWikiChannel = client.channels.cache.get(
+      channelIds.PROD.WIKI,
     ) as TextChannel
 
     schedule.scheduleJob('* * * *', async () => {
       console.log('Calling for new wikis 🚀')
 
-      await this.updates.sendUpdates(devChannel, ChannelTypes.DEV, this.DEV_URL)
+      await this.updates.sendUpdates({
+        channelId: devWikiChannel,
+        channelType: ChannelTypes.DEV,
+        url: `${this.DEV_URL}`,
+        updateType: UpdateTypes.WIKI,
+      })
 
-      await this.updates.sendUpdates(prodChannel, ChannelTypes.PROD, this.PROD_URL)
+      await this.updates.sendUpdates({
+        channelId: prodWikiChannel,
+        channelType: ChannelTypes.PROD,
+        url: `${this.PROD_URL}`,
+        updateType: UpdateTypes.WIKI,
+      })
+    })
 
+    schedule.scheduleJob('* * *', async () => {
```

Comment:
isnt this each second? i am not sure bcs of three *, just make sure its tested 👍🏻 we dont want to get banned from etherscan or discord, it should be per hour 👍🏻